src/pages/games/silverscreen.md

```diff
@@ -1,15 +1,3 @@
----
```

Comment:
The nice part about switching to Astro, is that I was able to port our markdown over with no additional setup needed. This page had (and continues to have) no layout. It's a WIP.